frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -673,7 +673,10 @@ class Worksheet extends React.Component {
                 let elementHeight = element.height();
                 let screenScrollTop = $(window).scrollTop();
                 let screenHeight = $(window).height();
-                let scrollIsAboveElement = elementOffsetTop + elementHeight - screenScrollTop >= 0;
+                // 170 is the sum of height of WorksheetHeader and NavBar.
+                // Since they block the user's view, we should take them into account when calculating whether the item is on the screen or not.
+                let scrollIsAboveElement =
+                    elementOffsetTop + elementHeight - screenScrollTop - 170 >= 0;
```

Comment:
Can we declare 170 in constants.js here? https://github.com/codalab/codalab-worksheets/blob/01b17c4499c0746af43433e922ff05cd860dd212/frontend/src/constants.js#L7

Reply:
Thanks for the advice. updated.